src/Esfa.Vacancy.Domain/Entities/ApprenticeshipSummary.cs

```diff
@@ -18,6 +18,8 @@ public class ApprenticeshipSummary
 
         public string ProviderName { get; set; }
 
+        public int ProviderUkprn { get; set; }
```

Comment:
Same naming thing applies here.

Reply:
We already had `ProviderName` on the entity so `ProviderUkprn` made sense to me.

Reply:
Only Name would be ambiguous, however Ukprn is not.

Reply:
If this is the return value from the endpoint, if my opinion counts, I would go with `Ukprn`

Reply:
OK. I'm outvoted here. I'll change the return value and the search parameter to `Ukprn`.